src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -376,12 +373,6 @@ private TypeParameterBounds GetBounds(ConsList<TypeParameterSymbol> inProgress,
             if (!currentBounds.IsSet(early))
             {
                 var constraintTypes = currentBounds.ConstraintTypes;
-                // PROTOTYPE(NullableReferenceTypes): Update to consider [NonNullTypes].
-                if (!ContainingModule.UtilizesNullableReferenceTypes)
-                {
-                    constraintTypes = constraintTypes.SelectAsArray(t => t.SetUnknownNullabilityForReferenceTypes());
```

Comment:
>SetUnknownNullabilityForReferenceTypes [](start = 75, length = 38)

:memo: `SetUnknownNullabilityForReferenceTypes` is still used, but very few places and I think they make sense (they will remain).